services/nomad/build/build-rsyncd.nomad

```diff
@@ -0,0 +1,79 @@
+job "build-rsyncd" {
+  type = "system"
```

Comment:
This should probably be a service with the right count.  System will try to put this on extra machines that aren't intended.